shared/lib/token-util.ts

```diff
@@ -0,0 +1,34 @@
+import abi from 'human-standard-token-abi';
```

Comment:
we should probably use `{ abiERC20 } from '@metamask/metamask-eth-abis'` here? Technically `fetchTokenBalance` could be calling to an ERC721 or ERC1155, but they all have the same `balanceOf` function so the function hash from the ABI is the same. Perhaps we should create a token standard agnostic ABI for this just to avoid confusion tho? cc @FrederikBolding

Reply:
We should replace all uses of `human-standard-token-abi` with `@metamask/metamask-eth-abis` in the repo. I can make a separate PR for this. Unless you'd like to @danjm

Reply:
please go ahead with a separate PR

Reply:
Actually at least for this instance you should probably just replace 
``` 
import abi from 'human-standard-token-abi';
```

with 

```
import { abiERC20 } from '@metamask/metamask-eth-abis'
```

and I can put up a PR for the other instances not added in this PR.

Reply:
okay, will do, thanks